src/NewTools-SettingsBrowser/StSettingsAbstractPresenter.class.st

```diff
@@ -23,6 +23,13 @@ StSettingsAbstractPresenter class >> appearanceIconContents [
 ]
 
+{ #category : 'accessing' }
+StSettingsAbstractPresenter class >> application [
```

Comment:
Shouldn't the class inherit from StPresenter to not have to do this kind of things?

Reply:
Thanks. I've updated the PR